test/tree-sitter-corpus.jl.faceup

```diff
@@ -0,0 +1,784 @@
+«x:# adjoint expressions»
+«x:# ==============================»
+
+[u, v]'
+A'[i]
+(x, y)'
+f'(x)
+«:julia-ts-quoted-symbol-face::a»'
+
+«x:# ==============================»
+«x:# juxtaposition expressions»
+«x:# ==============================»
+
+«c:1»x
+«c:2»v[i]
+«c:3»f(x)
+«c:4»df.a
+«c:5»«s:u"kg"»
+x'x
+«c:2»x^«c:2» - «c:.3»x
+«c:2»(x-«c:1»)^«c:2» - «c:3»(x-«c:1»)
+
+«x:# =============================»
+«x:# arrow function expressions»
+«x:# =============================»
+
+x -> x^«c:2»
+(x,y,z)-> «c:2»*x + y - z
+()->«c:3»
+() -> (sleep(«c:0.1»); i += «c:1»; l)
+a -> «v:a» = «c:2», «c:3»
+
+«x:# ==============================»
+«x:# boolean literals»
+«x:# ==============================»
+
+«c:true»
+«c:false»
+
+«x:# ==============================»
+«x:# integer number literals»
+«x:# ==============================»
+
+«c:0b01»
+«c:0o01234567»
+«c:0123456789»
+«c:123_456_789»
+«c:0x0123456789_abcdef_ABCDEF»
+
+«x:# ==============================»
+«x:# float number literals»
+«x:# ==============================»
+
+«c:0123456789.»
+«c:.0123456789»
+«c:0123456789.0123456789»
+
+«c:9e10»
+«c:9E-1»
+«c:9f10»
+«c:9f-1»
+
+«c:.1e10»
+«c:1.1e10»
+«c:1.e10»
+
+«c:0x0123456789_abcdef.ABCDEFp0»
+«c:0x0123456789_abcdef_ABCDEF.p-1»
+«c:0x.0123456789_abcdef_ABCDEFp1»
+
+«x:# ==============================»
+«x:# character literals»
+«x:# ==============================»
+
+«c:' '»
+«c:'o'»
+«c:'\t'»
+«c:'\uffff'»
+«c:'\U10ffff'»
+
+«x:# ==============================»
+«x:# string literals»
+«x:# ==============================»
+
+«s:""»
+«s:"\""»
+«s:"foo
+ bar"»
+«s:"this is a \"string\"."»
+«s:"""this is also a "string"."""»
+«v:band» = «s:"Interpol"»
+«s:"»«:julia-ts-string-interpolation-face:$»«D:band»«s: is a cool band"»
+«s:"»«:julia-ts-string-interpolation-face:$(»«c:2»«D:π»«:julia-ts-string-interpolation-face:)»«s: is a cool number"»
+«s:"cells interlinked within »«:julia-ts-string-interpolation-face:$(»«D:"cells interlinked whithin »«:julia-ts-string-interpolation-face:$(»«D:"cells interlinked whithin one stem"»«:julia-ts-string-interpolation-face:)»«D:"»«:julia-ts-string-interpolation-face:)»«s:"»
+
+«x:# ==============================»
+«x:# command string literals»
+«x:# ==============================»
+
+«s:`pwd`»
+«s:m`pwd`»
+«s:`cd »«:julia-ts-string-interpolation-face:$»«D:dir»«s:`»
+«s:`echo \`cmd\``»
+«s:```
+echo "\033[31mred\033[m"
+```»
+
+«x:# ==============================»
+«x:# non-standard string literals»
+«x:# ==============================»
+
+«x:# FIXME: \s shouldn't be an escape_sequence here»
+«v:trailing_ws» = «s:r"\s+$"»
+«v:version» = «s:v"1.0"»
+«s:K"\\"»
+
+«x:# ==============================»
+«x:# comments»
+«x:# ==============================»
+
+«x:# comment»
+«x:#= comment =#»
+«x:#=
+comment
+=#»
+«v:x» = «x:#= comment =#» «c:1»
+
+«x:#=
+nested #= comments =# =#»
+«x:#==#»
+
+«x:# ==============================»
+«x:# assignment operators»
+«x:# ==============================»
+
+«v:a» = b
+a .. b = a * b
```

Comment:
".." should be `font-lock-function-name-face`.

Reply:
Done.